Rework Installation into Distribution Channels (#131)
DevExpress/devextreme-documentation: concepts/Common/Distribution Channels/01 npm/10 Yarn.md

> All DevExtreme npm packages can be installed using Yarn:

Reviewer: ```suggestion
You can use Yarn to install all DevExtreme npm packages:
```
